conda_smithy/configure_feedstock.py

```diff
@@ -10,7 +10,7 @@
 import copy
 import hashlib
 import requests
-import time
+import json
```

Comment:
@hadim, 

The [requests is using the simplejson and not json](https://github.com/psf/requests/blob/master/requests/compat.py#L28-L31), when the former is available.


which makes  
https://github.com/conda-forge/conda-smithy/pull/1331/files#diff-3a2c4a048580db27ad9db93e2780b511R1361 
fails since the exception is simplejson.decoder.JSONDecodeError.

Replacing 

```python
import json
```
by 
```python
try:
	import simplejson as json
except:
	import json
```
or explicitly catching the second exception too finally solves the issue for me.